lvmsnapshotter.go

```diff
@@ -66,9 +66,19 @@ func NewSnapshotter(vgname string, lvpoolname string) (snapshots.Snapshotter, er
 	} else {
 		llog.Printf("Re-using the existing volume\n")
 	}
-	if _, err:= toggleactivateLV(vgname, metavolume, true); err != nil {
+	if _, err := toggleactivateLV(vgname, metavolume, true); err != nil {
 		return nil, errors.Wrap(err, "Unable to activate metavolume")
 	}
+	if fi, err := os.Stat(metaVolumeMountPath); os.IsNotExist(err) {
```

Comment:
Why is this change part of this commit?

Reply:
While we implemented kill, and unmount (which also removes the folder), this was added for completeness on creating and deleting the mount point.